icees_api/utils.py

```diff
@@ -40,3 +40,10 @@ def to_qualifiers2(feature):
         }
     else:
         return feature
+
+
+def association_matrix_empty(associations):
```

Comment:
The wording of this function and always using `not` seems a little weird. It could be straight english even: `if associations_have_feature_matrices(associations)`.